Automattic/sensei
# Add course theme variation body class

## Proposed Changes
* We've added a custom property in Course theme's theme json and the json setting of the variations as well. Here we parse the property value and add that to the body class. This will allow us to style the variations using CSS directly instead of having to write them in json files as strings.

## Testing Instructions


1. Checkout this branch of Course theme https://github.com/Automattic/themes/pull/7432
2. Select a variation of Course theme for your site from site editor and save
3. Reload from the frontend
4. Make sure the body element has the class `course-theme-variation-`[default/blue/gold/dark]
5. Check for all variations
6. Check with another block and non block theme to make sure nothing is broken

![Screenshot 2023-10-17 at 10 04 39 PM](https://github.com/Automattic/sensei/assets/6820724/b12a1e5d-83ce-494d-8050-5d380976ede3)

## Pre-Merge Checklist
